extensions/interactions/NumberWithUnits/directives/NumberWithUnits.js

```diff
@@ -135,9 +136,29 @@ oppia.directive('oppiaShortResponseNumberWithUnits', [
   }
 ]);
 
-// Rule evaluation for number with units (will be implemented in M1.3).
+// Rule evaluation for number with units.
 oppia.factory('numberWithUnitsRulesService', [
   'NumberWithUnitsObjectFactory', 'FractionObjectFactory',
   function(NumberWithUnitsObjectFactory, FractionObjectFactory) {
+    return {
+      IsEqualTo: function(answer, inputs) {
```

Comment:
Add a comment stating, if the input is exactly equals to answer.

Reply:
Done.